modules/opensearch/includes/opensearch.search.inc

```diff
@@ -142,6 +147,10 @@ function opensearch_search_search(TingSearchRequest $ting_query) {
     $options['facets'] = $ting_query->getFacets();
   }
 
+  if ($ting_query->getTermsPerFacet() !== NULL) {
+    $options['numFacets'] = $ting_query->getTermsPerFacet();
+  }
```

Comment:
This change does not seem to be related to quoting?

Reply:
Ah, one slipped trough, will fix